packages/local/src/cloud-image-prime.ts

```
@@ -0,0 +1,143 @@
+        cols: 80,
+        rows: 24,
+      });
+    });
+  } catch (error) {
+    primeError = error;
+    throw error;
+  } finally {
+    try {
+      await runner.shutdown();
+    } catch (shutdownError) {
+      if (primeError === undefined) throw shutdownError;
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)